test/controllers/api/v2/domains_controller_test.rb

```diff
@@ -413,7 +413,7 @@ def setup
       test 'user can create domain in default organization without need of specifying any organization' do
         domain = FactoryBot.build_stubbed(:domain)
         as_user @user do
-          post :create, params: { :domain => { :name => domain.name } }
+          post :create, params: { :domain => { :name => domain.name, :location_ids => [@loc1.org] } }
```

Comment:
I guess that should be `.id` instead of  `.org` everywhere, which is why jenkins fal